### ISSUE TITLE
Enforce Authentication for WPS-T front-end and ADES back-end

### ISSUE THREAD

**Issue by LucaCinquini** [unity-sds/unity-sps-prototype#60]

**Comment by mike-gangl** [comment]
I assume TEP backend is HySDS in this instance, correct? for the prototype, what auth is required and what TEP endpoints are we interacting with from the user perspective?

**Comment by LucaCinquini** [comment]
Yes TEP back-end is HySDS. That said, we need to discuss wether we want to expose any of the HySDS UIs to the user or not. At the very minimum, the WPS-T interface needs to be secured with authentication. I am not sure if we want  and/or have the resources to secure the HySDS UIs. Maybe we don't need them for the prototype.
